mtdata-stuff.py

```diff
@@ -53,3 +94,10 @@ def read_dataset(did: str):
 @app.get("/datasets/{did}/sample")
```

Comment:
This is duplicated with the function above. I assume that this is part of the javascript interface. @jelmervdl is that how it's supposed to work?